### ISSUE TITLE
[Source Request][French] Manga-ScanTrad

### ISSUE THREAD

**Issue by pixelroll** [inorichi/tachiyomi-extensions#3716]
Source Website

- Name: Manga Scantrad

- Link: https://manga-scantrad.net/

- Language(s): French

Other Details :
- it's based on the same structure (template) as novel france (but novel france doesn't work anymore), so it's almost the same site.

- This is the website of a small French team, which translates the least advanced webtoons in France.
For example Star Martial God Technique which is translated up to chapter 150 in French on mangadex and all other French sources available on Tachiyomi.
On this site it is available up to chapter 335 which is the last chapter released.
In short, all webtoons available on this site get a very fast and high quality translation.
Thanks
Sincerely Pixelroll

**Comment by pixelroll** [comment]
https://manga-scantrad.net/

**Comment by pixelroll** [comment]
what does factory source mean ?
